driver/normalizer/annotation.go

```diff
@@ -119,9 +81,9 @@ var AnnotationRules = On(Any).Self(
 		On(HasInternalType(pyast.Div)).Roles(OpDivide),
 		On(HasInternalType(pyast.Mod)).Roles(OpMod),
 		// TODO: currently without mapping in the UAST
-		//On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide),
-		//On(HasInternalType(pyast.Pow)).Roles(???),
-		//On(HasInternalType(pyast.MatMult)).Roles(???),
+		On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide, Incomplete),
+		On(HasInternalType(pyast.Pow)).Roles(OpMultiply, Incomplete),
```

Comment:
I think that's how we defined:
* `Unannotated` role should be used to detect problems with the annotation process. The driver developer should aim to have none of these that they know of.
* `Incomplete` role should be used to mark nodes that aren't completely annotated. Not completely annotated includes _not annotated at all_, but not _incorrectly annotated_, which should never be done.

That's how I understood it, maybe @mcuadros can clearify it better.